OneWelcomeExampleApp/ErrorMappings/ErrorMapper.swift

```diff
@@ -16,23 +16,23 @@
 class ErrorMapper {
     func mapError(_ error: Error, pinChallenge: PinChallenge? = nil, customInfo: CustomInfo? = nil) -> AppError {
         switch error.domain {
```

Comment:
switch ErrorDomains(rawValue: error.domain)

Reply:
`ErrorDomains` is not yet a propper enum
```
// TODO: To be replaced with proper enum in pure Swift
// swiftlint:disable line_length
@objcMembers
public class ErrorDomains: NSObject {